README.md

```diff
@@ -10,27 +9,17 @@
 
 Node/Browser Javascript library for ArcGIS
 
-## Install
-
 ```
 npm install arcgis
 ```
 
-## Usage
-
```

Comment:
agreed on all points.

@nikolaswise already has a jump start on this in #68. so if he wants to rip off my fixes for the `package.json` license syntax and website url update, we'll just close this one.